src/utils/time-converter.js

```diff
@@ -0,0 +1,17 @@
+export const timeConverter = (timeString)=> {
+    const date = new Date(timeString);
+  
+    const year = date.getFullYear();
+    const month = String(date.getMonth() + 1).padStart(2, '0');     const day = String(date.getDate()).padStart(2, '0');
+   
+    const hours = String(date.getHours()).padStart(2, '0');
+    const minutes = String(date.getMinutes()).padStart(2, '0');
+    const seconds = String(date.getSeconds()).padStart(2, '0');
+  
+    const visualTime = `${year}-${month}-${day} ${hours}:${minutes}:${seconds}`;
+  
+    return visualTime;
+  }
```

Comment:
As we have a utils function here, we should also have a test file for it 👍

Reply:
Thank you for the feedback John. i have finally written tests for my utils functions.  It was an absolute headache figuring out hot to configure jest for ES6 modules.